torch/ao/quantization/fuse_modules.py

```diff
@@ -160,8 +160,8 @@ def fuse_modules(model, modules_to_fuse, inplace=False, fuser_func=fuse_known_mo
         modules_to_fuse,
         is_qat=False,
         inplace=inplace,
-        fuser_func=fuse_known_modules,
-        fuse_custom_config_dict=None)
+        fuser_func=fuser_func,
+        fuse_custom_config_dict=fuse_custom_config_dict)
```

Comment:
`fuse_custom_config_dict` is deprecated I think, we plan to add this to https://pytorch.org/docs/master/generated/torch.ao.quantization.backend_config.BackendConfig.html#torch.ao.quantization.backend_config.BackendConfig instead, and this part is not yet been designed

Reply:
Got it. Removed changes to fuse_custom_config_dict in an updated diff.